actor/manager/manager.go

```diff
@@ -0,0 +1,240 @@
+		serializer: serializer,
+	}, actorErr.Success
+}
+
+// RegisterActorImplFactory registers the action factory f
+func (m *DefaultActorManager) RegisterActorImplFactory(f actor.Factory) {
+	m.factory = f
+}
+
+// getAndCreateActorContainerIfNotExist will
+func (m *DefaultActorManager) getAndCreateActorContainerIfNotExist(actorID string) ActorContainer {
+	val, ok := m.activeActors.Load(actorID)
+	if !ok {
+		m.activeActors.Store(actorID, NewDefaultActorContainer(actorID, m.factory(), m.serializer))
+		val, _ = m.activeActors.Load(actorID)
+	}
+	return val.(ActorContainer)
+}
+
+// InvokeMethod to invoke local function by @actorID, @methodName and @request request param
+func (m *DefaultActorManager) InvokeMethod(actorID, methodName string, request []byte) ([]byte, actorErr.ActorErr) {
+	if m.factory == nil {
+		return nil, actorErr.ErrActorFactoryNotSet
+	}
+	var replyv reflect.Value
+	returnValue, aerr := m.getAndCreateActorContainerIfNotExist(actorID).Invoke(methodName, request)
+	if aerr != actorErr.Success {
+		return nil, aerr
+	}
+	if len(returnValue) == 1 {
+		return nil, actorErr.Success
+	}
+
+	var retErr interface{}
+	if len(returnValue) == 2 {
+		replyv = returnValue[0]
+		retErr = returnValue[1].Interface()
+	}
+
+	if retErr != nil {
+		panic(retErr)
+	}
+	rspData, err := json.Marshal(replyv.Interface())
+	if err != nil {
+		return nil, actorErr.ErrActorInvokeFailed
+	}
+	m.getAndCreateActorContainerIfNotExist(actorID).GetActor().SaveState()
+	return rspData, actorErr.Success
+}
+
+// DetectiveActor removes actor from actor manager
+func (m *DefaultActorManager) DetectiveActor(actorID string) actorErr.ActorErr {
+	_, ok := m.activeActors.Load(actorID)
+	if !ok {
+		return actorErr.ErrActorIDNotFound
+	}
+	m.activeActors.Delete(actorID)
+	return actorErr.Success
+}
+
+// InvokeReminder invoke reminder function with given params
+func (m *DefaultActorManager) InvokeReminder(actorID, reminderName string, params []byte) actorErr.ActorErr {
+	if m.factory == nil {
+		return actorErr.ErrActorFactoryNotSet
+	}
+	reminderParams := &api.ActorReminderParams{}
+	if err := json.Unmarshal(params, reminderParams); err != nil {
+		fmt.Println("unmarshal reminder param error = ", err)
+		return actorErr.ErrRemindersParamsInvalid
+	}
+	targetActor, ok := m.getAndCreateActorContainerIfNotExist(actorID).GetActor().(actor.ReminderCallee)
+	if !ok {
+		return actorErr.ErrReminderFuncUndefined
+	}
+	targetActor.ReminderCall(reminderName, reminderParams.Data, reminderParams.DueTime, reminderParams.Period)
+	return actorErr.Success
+}
+
+// InvokeTimer invoke timer callback function with given  params
+func (m *DefaultActorManager) InvokeTimer(actorID, timerName string, params []byte) actorErr.ActorErr {
+	if m.factory == nil {
+		return actorErr.ErrActorFactoryNotSet
+	}
+	timerParams := &api.ActorTimerParam{}
+	if err := json.Unmarshal(params, timerParams); err != nil {
+		fmt.Println("unmarshal reminder param error = ", err)
```

Comment:
change like `log.Printf("failed to unmarshal reminder param, err: %v ", err)`, same else.